src/test/java/com/microsoft/azure/kusto/kafka/connect/sink/FileWriterTest.java

```diff
@@ -121,9 +121,9 @@ public void testGzipFileWriterFlush() throws IOException, InterruptedException {
 
         final int MAX_FILE_SIZE = 128 * 2;
 
-        Consumer<FileDescriptor> trackFiles = (FileDescriptor f) -> files.put(f.path, f.rawBytes);
+        Consumer<FileProperties> trackFiles = (FileProperties f) -> files.put(f.path, f.rawBytes);
 
-        Supplier<String> generateFileName = () -> Paths.get(path, java.util.UUID.randomUUID().toString()).toString();
+        Supplier<String> generateFileName = () -> Paths.get(path, java.util.UUID.randomUUID().toString()).toString() + "csv.gz";
```

Comment:
shouldn't this be configurable?

Reply:
its just for testing ..
we provide here the path ourselves ..